app/src/main/res/layout/activity_main.xml

```diff
@@ -47,6 +47,17 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.606" />
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="User Guide: \nPlease enter 3 to 8 integers which you would like sorted.\nE.g. '5 8 1' Also please only enter integers between 0-9."
```

Comment:
2 things here: 
1) "User Guide:" is sort of a reserved term  for a document that is provided to users on how to use an application. I don't think you really need this. Especially because if you only have some text and then immediately following a textbox. If you want to keep a subtitle over this, I think 'Directions:' would be more appropriate. 
2) Instead of adding a second sentence, it would be cleaner to have it all in one sentence. Please update this description from "3 to 8 integers" to be "3 to 8 single digit integers"